apps/web/src/utils/viem.ts

```diff
@@ -29,6 +29,6 @@ export const viemClients = CHAINS.reduce((prev, cur) => {
   }
 }, {} as Record<ChainId, PublicClient>)
 
-export const getViemClients = ({ chainId }: { chainId?: ChainId }) => {
+export const getViemClients = ({ chainId }: { chainId?: ChainId }): PublicClient => {
```

Comment:
hmm if the chainId is possibly `undefined` then we may need to return undefined as well
```suggestion
export const getViemClients = ({ chainId }: { chainId?: ChainId }): PublicClient | undefined => {
```